vscode-lean4/src/infoview.ts

```diff
@@ -10,8 +10,50 @@ import { LeanClient } from './leanclient';
 import { getInfoViewAllErrorsOnLine, getInfoViewAutoOpen, getInfoViewAutoOpenShowGoal,
     getInfoViewFilterIndex, getInfoViewStyle, getInfoViewTacticStateFilters } from './config';
 import { Rpc } from './rpc';
-import { PublishDiagnosticsParams } from 'vscode-languageclient';
 import * as ls from 'vscode-languageserver-protocol'
+import { LanguageClient } from 'vscode-languageclient/node';
+
+interface RpcConnectParams {
```

Comment:
Can we put these in `lean4-infoview/src/lspTypes.ts`?

Reply:
That's a good idea.